up-and-running/auth/ldap.md

```diff
@@ -0,0 +1,86 @@
+# LDAP Authentication
+
+npm Enterprise supports most LDAP corporate directories, including Active Directory and OpenLDAP.
+
+It's important to note that, when using LDAP integration, your npm Enterprise instance(s) should typically be running on the same subnet as your LDAP servers.
+
+Note that you can also use LDAP integration for logging into the admin console (port `:8800`) as well. If you'd like to set that up, visit the `/create-password` page once you're logged in, choose the `LDAP`, and enter the settings just as you would below.
+
+## Configuration Fields
+
+### Server Settings
```

Comment:
It's a little weird seeing three headings in a row, does this look nice when rendered in GitBook? I like the structure.